### PR TITLE
feat(associate): added basic associate game functionality

### PR DESCRIPTION
**Pull request by namiwa**
-  Added basic associate game page only

- This includes other components which was developed but not used in the main associate game

### CODE REVIEW

File: src/routes/associate/Game/Game.tsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
+import React, { useState } from 'react';
+import { BrowserView, MobileView } from 'react-device-detect';
+import { DndProvider } from 'react-dnd';
```
Comment (nwjbrandon) [inline src/routes/associate/Game/Game.tsx]: React dnd does not work on mobile. If cannot figure out, I will take over when you change the layout of the game

File: src/routes/associate/Game/GameImages.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import Sushi from '../../../assets/images/associate/food/sushi.svg';
+
+import Bike from '../../../assets/images/associate/vehicles/bike.svg';
+import Scooter from '../../../assets/images/associate/vehicles/scooter.svg';
+import Train from '../../../assets/images/associate/vehicles/train.svg';
+import Truck from '../../../assets/images/associate/vehicles/truck.svg';
+import Wheel from '../../../assets/images/associate/vehicles/wheel.svg';
+
+import Carpet from '../../../assets/images/associate/furniture/carpet.svg';
+import Chair from '../../../assets/images/associate/furniture/chair.svg';
```
Comment (nwjbrandon) [inline src/routes/associate/Game/GameImages.tsx]: Use absolute imports, looks neater

File: src/routes/associate/Game/Game.tsx [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+  const [displayGame] = useState(setUpGame());
+
+  const DisplayItems: React.FC<GameComponentProp> = ({ store }) => {
+    return (
+      <div className="left">
+        {store.map(val => {
+          return (
+            <div key={val.img}>
+              <div>
+                <Item
```
Comment (nwjbrandon) [inline src/routes/associate/Game/Game.tsx]: Currently the game is has two columns where the left is match to right. Can try having one big image and four small image at the bottom. So one of the four image at the bottom is map to the big image?
demo

File: src/routes/associate/Game/GameImages.tsx (hunk 2 above)
Comment (namiwa) [reply src/routes/associate/Game/GameImages.tsx]: Noted, will be added in commit 11f07a9

File: src/routes/associate/Game/Game.tsx (hunk 3 above)
Comment (namiwa) [reply src/routes/associate/Game/Game.tsx]: Noted, will be added in commit a42a38c

File: src/routes/associate/Game/Game.tsx (hunk 1 above)
Comment (namiwa) [reply src/routes/associate/Game/Game.tsx]: Noted, will continue working on the previously implement container using react-draggable

**Pull request merged**